.github/PROCESS.md

```diff
@@ -0,0 +1,15 @@
+# The Process
+
+This file contains some process notes for FerretDB Inc. engineers and long-term contributors.
+If you are a community contributor, feel free to ignore all this!
+We will do all that for you so you can focus on what is really interesting for you.
+
+1. Pull request title should be accurate and descriptive as it is used in the generated changelog.
```

Comment:
let's highlight somehow that this is about **title** and point 2 is about desciption.

Reply:
What do you mean? Highlight by using _italics_ or **bold**?

Do you tend to read files like that on GitHub or locally?

Reply:
Both, on github and locally.

Reply:
bold.